tests/conftest.py

```diff
@@ -17,15 +17,31 @@
 root = Path(os.getcwd())
 pyrootutils.set_root(path=root, dotenv=True, pythonpath=True)
 
-experiments_names = [
+experiments_require_torchvision = [
     "CIFAR10_CNN",
     "CIFAR10_CNN_Adv",
-    "ImageNet_Timm",
     "COCO_TorchvisionFasterRCNN",
     "COCO_TorchvisionFasterRCNN_Adv",
     "COCO_TorchvisionRetinaNet",
 ]
 
+experiments_require_torchvision_and_timm = [
+    "ImageNet_Timm",
+]
+
+if os.getenv("CI") == "true":
```

Comment:
Is this environment variable set by Github?

Reply:
Yes, it is set by many CI systems: https://stackoverflow.com/a/75223617